Networking/Networking/ApplicationPassword/ApplicationPasswordStorage.swift

```diff
@@ -7,8 +7,8 @@ struct ApplicationPasswordStorage {
     ///
     private let keychain: Keychain
 
-    init(keychain: Keychain = Keychain(service: KeychainServiceName.name)) {
-        self.keychain = keychain
+    init(keychain: Keychain? = nil) {
+        self.keychain = keychain ?? Keychain(service: KeychainServiceName.name)
```

Comment:
Even though we set a value to it in the initializer, the optional parameter can lead to a misunderstanding that the keychain is not required for the storage. How about using the default value in both `ApplicationPasswordStorage` and `DefaultApplicationPasswordUseCase` initializers?

Reply:
I followed this approach as I had to make the keychain service name public otherwise.

Done in cae21d0cf1e17fc2544a383654a7696783f282dd and 82277ce94bca53724f2d9ae4ae2109d7d9a6e77a